Index: Data/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Data/build.gradle b/Data/build.gradle
--- a/Data/build.gradle	(revision f5bca03091d253d9c4c0d089c4917f4ee4a275da)
+++ b/Data/build.gradle	(date 1657719377569)
@@ -2,6 +2,7 @@
     id 'com.android.library'
     id 'org.jetbrains.kotlin.android'
     id 'kotlin-kapt'
+    id 'com.google.devtools.ksp' version '1.7.10-1.0.6'
 }
 
 android {
@@ -33,9 +34,9 @@
 dependencies {
 
 
-    implementation("androidx.room:room-runtime:2.3.0")
-    kapt("androidx.room:room-compiler:2.3.0")
-    implementation "androidx.room:room-ktx:2.3.0"
+    implementation("androidx.room:room-runtime:2.4.2")
+    ksp("androidx.room:room-compiler:2.4.2")
+    implementation "androidx.room:room-ktx:2.4.2"
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1")
     implementation("androidx.lifecycle:lifecycle-livedata-ktx:2.6.0-alpha01")
     implementation 'androidx.core:core-ktx:1.8.0'
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision f5bca03091d253d9c4c0d089c4917f4ee4a275da)
+++ b/build.gradle	(date 1657719226289)
@@ -11,7 +11,7 @@
 plugins {
     id 'com.android.application' version '7.2.1' apply false
     id 'com.android.library' version '7.2.1' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.10' apply false
+    id 'org.jetbrains.kotlin.android' version '1.7.10' apply false
 }
 
 task clean(type: Delete) {
